app/components/utils/next-button/component.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+import { inject } from '@ember/service';
+
+export default Component.extend({
+  router: inject(),
+  actions: {
+    async onRowClick() {
+      const account = await this.row.content.account;
+      const user = await account.get('user');
+      const userId = await user.get('id');
+      this.router.transitionTo('settings.users.user', userId);
+    },
+  }
+});
```

Comment:
This component is in /utils but can only be used in the user settings ?
Either rename the component for its specific functionality (next-user-button) or preferably move it to components/settings/user folder (create the folders)